packages/jupyter-ai/jupyter_ai/extension.py

```diff
@@ -406,3 +407,10 @@ def _init_persona_manager(self, ychat: YChat) -> Optional[PersonaManager]:
             self.log.exception(e)
         finally:
             return persona_manager
+
+    def _link_jupyter_server_extension(self, server_app):
```

Comment:
nit: `-> None`

Not sure the `server_app` has a typing, but it would be nice to add args typing if it does.